### DISCUSSION TITLE
Pages section in Dashboard

### DISCUSSION THREAD

**Discussion by tlosbar** [saleor/saleor#8160] [Ideas]
As pages have more powers in the next release 3.0, why not moving the Pages configuration higher in the Dashboard? Near the General sections like Catalog, Orders, Translations.

I think it is one of the best new features in the next update. It can be a minor alternative to other Headless CMS like Contentful or Prismic. The Shop owner will not have to go to another service and will have the possibility to directly edit page details for its products.

**Comment by karolkielecki** [comment]
@tlosbar Hi Thomas! Thanks for your suggestion! We're also very excited about this feature. That's our plan to move `Pages` near the general sections. Currently, we just have other priorities.

Cheers!
